docs/pages/installation/amazon-ecs.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Installing Teleport on Amazon ECS using Terraform
+description: How to install and run Teleport agents on Amazon ECS
+labels:
+ - platform-wide
+ - reference
+---
+
+# Step 1/3. ....
```

Comment:
We should replace this H1 with one or more introductory paragraphs that indicate to a user when/why they should follow this giude.